src/pages/DetailPage/Components/DishData.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import './DishData.scss';
+import { INFO_DATA } from './INFORMATION_DATA';
+class DishData extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      data: {},
+    };
+  }
+
+  componentDidMount() {
+    fetch('./data/data.json')
```

Comment:
```suggestion
    fetch('/data/data.json')
```
. 으로 시작하는 상대경로가 아니라 절대경로로 적어주셔야 합니다!
data.json이라는 파일명보다는 어떤 데이터인지 드러나는 파일명으로 바꿔주세요!